### PR TITLE
SLO roles privileges changed

### PR DESCRIPTION
**Pull request by eedugon**
## Description
As described at https://github.com/elastic/observability-docs/issues/4229 in 8.16 the SLO permissions have changed, and also the builtin `editor` role is an option to grant privileges for SLOs.

This PR updates the `slo-privileges.asciidoc` and its associated images.
It also adds a minor change in serverless doc, as the required role to create SLOs is `Editor` and not `Admin`.

### Documentation sets edited in this PR

_Check all that apply._

- [X] Stateful (`docs/en/observability/*`)
- [x] Serverless (`docs/en/serverless/*`)
- [ ] Integrations Developer Guide (`docs/en/integrations/*`)
- [ ] None of the above

### Related issue
Closes https://github.com/elastic/observability-docs/issues/4229

## Checklist

- [ ] Product/Engineering Review
- [ ] Writer Review

### Follow-up tasks

_Select one._

* This PR does _not_ need to be ported to another doc set because:
  - [ ] The concepts in this PR only apply to one doc set (serverless _or_ stateful)
  - [x] The PR contains edits to both doc sets (serverless _and_ stateful)
* This PR needs to be ported to another doc set:
  - [ ] Port to stateful docs: \<link to PR or tracking issue>
  - [ ] Port to serverless docs: \<link to PR or tracking issue>

### CODE REVIEW

**Comment by eedugon** [comment]
Thanks @bmorelli25 for your comments and suggestions here!

**Review by dominiqueclarke — DISMISSED** [review]
LGTM

File: docs/en/observability/slo-privileges.asciidoc [hunk 1]
```diff
@@ -7,10 +7,21 @@
...
 IMPORTANT: To create and manage SLOs, you need an {subscriptions}[appropriate license].
 
-You can create the following roles for your SLOs:
+You can enable access to SLOs in two different ways:
 
-* <<slo-all-access,*SLO All*>> — Create, edit, and manage SLOs and their historical summaries.
-* <<slo-read-access,*SLO Read*>> — See SLOs and their historical summaries.
+* Creating the following roles, depending on the type of access needed:
+** <<slo-all-access,*SLO Editor*>> — Create, edit, and manage SLOs and their historical summaries.
+** <<slo-read-access,*SLO Viewer*>> — See SLOs and their historical summaries.
```
Comment (alaudazzi) [inline docs/en/observability/slo-privileges.asciidoc:14]:
```suggestion
** <<slo-read-access,*SLO Viewer*>> — Check SLOs and their historical summaries.
```
Replace visual clues with more accessible terms.
For more info, refer to [How to create accessible web content](https://docs.google.com/document/d/1IGuMf16-cwxx8BX1Z9PWsFWsTPxicwTkX_r7UOhjd4E/edit#heading=h.iykpo698d18b).

File: docs/en/observability/slo-privileges.asciidoc [hunk 2]
```diff
@@ -7,10 +7,21 @@
...
+
+* Using the `editor` {ref}/built-in-roles.html[built-in role]. This role grants full access to all features in {kib} (including the {observability} solution)
+and read-only access to data indices. Users assigned to this role can create, edit, and manage SLOs.
++
+[NOTE]
+====
+The `editor` {ref}/built-in-roles.html[built-in role] will grant write access
+to _all_ {kib} apps. If you want to limit access to the SLOs only, you will have
+to manually create and assign the mentioned roles.
+====
```
Comment (alaudazzi) [inline docs/en/observability/slo-privileges.asciidoc:24]:
```suggestion
====
The `editor` {ref}/built-in-roles.html[built-in role] grants write access
to _all_ {kib} apps. If you want to limit access to the SLOs only, you have
to manually create and assign the mentioned roles.
====
```
Limiting the use of future tense and preferring the present tense where possible makes the reading lighter.

File: docs/en/observability/slo-privileges.asciidoc [hunk 3]
```diff
@@ -20,37 +31,38 @@ To create a role:
...
 
-Set the following privileges for the SLO All role:
+Set the following privileges for the SLO Editor role:
 
-. Under *Cluster privileges* in the *Elasticsearch* section, add `manage_transform` and `manage_ingest_pipelines`.
-. Under *Index privileges*, add `.slo-*` to the *Indices* field and `all` to the *Privileges* field.
+. Under *Index privileges* in the *Elasticsearch* section, add `.slo-observability-*` to the *Indices* field and `read`, `view_index_metadata`, `write`, and `manage` to the *Privileges* field.
 . Click *Add index privilege*.
-. Add indices you plan to create SLOs for to the *Indices* field, and add *read* and *view_index_metadata* to the *Privileges* field. The following example shows `logs-*`, but you can specify any indices.
+. Add indices you plan to create SLOs for to the *Indices* field, and add `read` and `view_index_metadata` to the *Privileges* field. The following example shows `logs-*`, but you can specify any indices.
```
Comment (alaudazzi) [inline docs/en/observability/slo-privileges.asciidoc:40]:
```suggestion
. In the *Indices* field, add the indices for which you plan to create SLOs. Then, add `read` and `view_index_metadata` to the *Privileges* field. The following example shows `logs-*`, but you can specify any indices.
```

**Review by alaudazzi — APPROVED** [review]
I left a few minor editing suggestions, otherwise LGTM.

**Review by alaudazzi — APPROVED** [review]
LGTM

**Comment by eedugon** [comment]
quick question here @alaudazzi and @bmorelli25 : do we need a backport to 8.16 here or was this unneeded at that time?

**Comment by eedugon** [comment]
I think it does, I don't find the commit of this PR in `8.16` and I'm able to find it in `main` and `8.x`.... creating the backport now...

**Pull request merged**